cfme/tests/cloud_infra_common/test_events.py

```diff
@@ -34,7 +34,7 @@ def vm_crud(provider, setup_provider_modscope, small_template_modscope):
 
 
 @pytest.mark.uncollectif(BZ(1491576, forced_streams=['5.7']).blocks, 'BZ 1491576')
-@pytest.mark.meta(blockers=[1238371], automates=[1238371])
+@pytest.mark.meta(blockers=[JIRA('RHCFQE-4749')])
```

Comment:
**Required/Question**: Does this issue touch only rhv41 provider?

Reply:
It's really hard for me to say since I do not work with any other provider.

Reply:
yes, it is relevant only for rhv41. Please don't put Jira. I'm raising BZ and will add it to test soon

Reply:
@izapolsk alrgiht, this PR is not really required, so closing. Thank you for looking into it so quickly